bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/InstallToolsHandler.java

```diff
@@ -83,6 +95,38 @@ protected IStatus run(IProgressMonitor monitor)
 		installToolsJob.schedule();
 
 	}
+	
+	private void copyOpenOcdRules()
+	{
+		if (Platform.getOS().equals(Platform.OS_LINUX))
+		{
+			if (!IDFUtil.getOpenOCDLocation().equalsIgnoreCase(StringUtil.EMPTY))
```

Comment:
Can we combine these two if statements to avoid nested if statements?

```
if (Platform.getOS().equals(Platform.OS_LINUX)) && !IDFUtil.getOpenOCDLocation().equalsIgnoreCase(StringUtil.EMPTY)) 
{
 //copy rules
}
```

Reply:
Done